### ISSUE TITLE
Transmutation Chamber dimension issue

### ISSUE THREAD

**Issue by 1JackBlack1** [pkmnfrk/equivalentintegrations#39]
### tldr
Transmutation chamber not working correctly across dimensions.

### Setup
I have a transmutation chamber hooked up to an ME system from applied energistics set to be the highest priority storage.
The entire ME system is in the overworld inside loaded chunks, with most of it in 1 chunk (a single drive extends into the next chunk, as does some of the storage draws it is hooked up to).
The transmutation chamber has the alchemical algorithms and efficiency catalysts installed.
I access it via a wireless crafting terminal from wct, with an infinity booster card from ae2wtlib.

### Problem
When in the overworld it works as expected. When I insert an item into the ME system, it goes into the chamber and is learnt by the system so I can pull out as many as my EMC allows.
When in another dimension (I have tested in nether and the hunting dimension) the item does not go into the transmutation chamber and instead is being stored elsewhere; however I can still pull out items from the chamber.

I am playing on Project Ozone 3, with EI, Project E, Project Ex and Mike's Mod Lib updated to the latest version. This also happened before updating these mods.

**Comment by 1JackBlack1** [comment]
To confirm:
I made a single player creative test world with a similar setup all in one chunk, with a creative energy cell, and the same thing happens. It all works fine in the over-world with items being learnt as they are inserted for the first time with me then able to take out as many as I want.
When I go to the nether the items not already learnt end up on the disk and I can only take out a finite amount. Items that have been learnt go into the chamber just fine and I can take items out of the chamber.

There is nothing in latest.log when this happens and the only thing in debug.log is repeated spamming of Returned EMC value.

While in the nether I can put the item into the transmutation tablet to learn the item and then it inserts into the chamber just fine.

**Comment by pkmnfrk** [comment]
Hi, sorry for not responding, I've been busy starting a new job, so I haven't had a chance to look at this yet.

It is very weird that moving to another dimension would cause trouble. ProjectE is very specific that a player who is Offline cannot be interacted with (you can't change EMC, they can't learn things, etc). But, being in the Nether is not the same as being offline.

I will have to look into this, but I have no idea right now.

**Comment by Plasman888** [comment]
If you install the projectEX mod, it has personal EMC links you can use some of them, I use the compact refined i=one instead of the transmutation disassembler and your set up should work properly after you play around with the settings.

**Comment by Toskin** [comment]
Can confirm this input problem still happens. Tried it with Refined Storage, Dimensional Doors and Random Things Spectre Key, and I can't force Chamber to learn new things when I'm not in same dimension. For now I just throw cell into IO block from time to time to export new items into EMC system.

**Comment by Toskin** [comment]
> If you install the projectEX mod, it has personal EMC links you can use some of them, I use the compact refined i=one instead of the transmutation disassembler and your set up should work properly after you play around with the settings.

So, if you hook up storage bus to Link, it can see items your EMC system learned and extract them?
